iris_markdown_EH.Rmd

```diff
@@ -0,0 +1,123 @@
+---
+title: "Ethan's Iris Analysis in RMarkdown"
+author: "Ethan Hill"
+date: "2020-01-16"
+output: html_document
+---
+
+Hello! Here are the plots and the code for the plots I have generated with the Iris dataset. I have also gone ahead and included the workflow I used to get the final plots...
+
+
+It is always a good habit to set your working directory:
+
+```{r}
+setwd("C:/Users/hille/Documents/Rbootcamp_S2020/Iris-repo/")
+```
```

Comment:
True, but donʻt forget to comment it out in any script. You can also save an empty x.Rdata file (in windows) or x.R (in mac) in the appropriate directory and start from there.

Reply:
Unfortunately, this also didnʻt get fixed. Normally, in a coding class you would get a zero for a program that doesnʻt even run.  -1